app/views/registration/_form.erb

```diff
@@ -0,0 +1,71 @@
+<% title _("Register Host") %>
+<%= form_tag({controller: 'registration', action: 'create' }, method: :post, class: 'form-horizontal well') do %>
```

Comment:
I guess we can't easily use traditional form helpers, since we don't have model here.